graphql/resolve/resolver.go

```diff
@@ -480,15 +480,15 @@ func (r *RequestResolver) Resolve(ctx context.Context, gqlReq *schema.Request) (
 			},
 		},
 	}
-	defer func() {
-		// Panic Handler for mutation. This ensures that the mutation which causes panic
-		// gets logged in Alpha logs. This panic handler overrides the default Panic Handler
-		// used in recoveryHandler in admin/http.go
-		api.PanicHandler(
-			func(err error) {
-				resp.Errors = schema.AsGQLErrors(schema.AppendGQLErrs(resp.Errors, err))
-			}, gqlReq.Query)
+	// Panic Handler for mutation. This ensures that the mutation which causes panic
+	// gets logged in Alpha logs. This panic handler overrides the default Panic Handler
+	// used in recoveryHandler in admin/http.go
+	defer api.PanicHandler(
```

Comment:
Maybe we should also add a comment about why we can't merge this defer with the other defer. That would be really helpful for anyone else to get context.